discogs_alert/scrape.py

```diff
@@ -49,22 +49,12 @@ def scrape_listings_from_marketplace(response_content: str) -> da_types.Listings
 
         item_condition_para = item_desc_cell.find("p", class_="item_condition")
 
-        # extract media condition (via ignoring the context of the new media condition tooltip)
-        media_condition_span = item_condition_para.find_all("span")[2]
-        media_condition_tooltip_span = media_condition_span.find("span", class_="has-tooltip")
-        media_condition_tooltip_span.replace_with("")
-        media_condition = media_condition_span.text.strip()
-        listing["media_condition"] = da_types.CONDITION_PARSER[media_condition]
-
-        # extract sleeve condition
-        sleeve_condition_spans = item_condition_para.find("span", class_="item_sleeve_condition")
-        if sleeve_condition_spans is not None:
-            sleeve_condition = sleeve_condition_spans.contents[0].strip()
-            sleeve_condition = da_types.CONDITION_PARSER[sleeve_condition]
-        else:
-            # sometimes the sleeve condition isn't listed => ungraded
-            sleeve_condition = da_types.CONDITION.NOT_GRADED
-        listing["sleeve_condition"] = sleeve_condition
+        # extract conditions of media (always listed) and sleeve (optional)
+        conditions = list(filter(None, map(da_types.CONDITION_PARSER.get, item_condition_para.stripped_strings)))
```

Comment:
```suggestion
        conditions = (da_types.CONDITION_PARSER.get(s) for s in item_condition_para.stripped_strings)
        conditions = [c for c in conditions if c is not None]
```
single-pass generator approach